flake.nix

```diff
@@ -1,6 +1,8 @@
 {
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05";
+    nixpkgs.url = "https://flakehub.com/f/NixOS/nixpkgs/0.2305.*.tar.gz";
+
+    flake-utils.url = "https://flakehub.com/f/numtide/flake-utils/0.1.*.tar.gz";
```

Comment:
Aside from being `follows`'d in a couple other inputs and still present in the output function's arguments, this doesn't seem used anywhere.
```suggestion
```

Reply:
Hmm, weird oversight on my part